libmscore/score.cpp

```diff
@@ -1469,7 +1469,7 @@ void Score::removeElement(Element* element)
             measures()->remove(mb);
             System* system = mb->system();
             Page* page = system->page();
-            if (element->isVBox() && system->measures().size() == 1) {
+            if ((element->isVBox() || element->isTBox()) && system->measures().size() == 1) {
```

Comment:
If this code should be executed for all vertical frames and classes that inherit VBox, you forgot FBox

Reply:
In that case, then maybe we want an isVBoxBase() function that returns true for VBox and all of its descendants. See isTextBase(), for example.

Reply:
Unless it would be more appropriate to redefine the isVBox() function to also return true for all of its descendants.

Reply:
I don't know. What's an FBox? Try adding one to the first measure in a score, and deleting it. Does the same bug occur? If so, then yes, it needs handling. Otherwise, no, it hasn't been forgotten, and is handled elsewhere.

Reply:
I've made a isVBoxBase() function, if you want to use it. It is in #4705 in `libmscore/scoreElement.h`

Reply:
I think it is needed. `FBox`es are weird, in that they don't have a user-visible, unprintable representation on-screen. Thus the only way I could get one selected is if I insert it and not click anywhere else. If I then press the Del key, it is deleted, but the space created for it on screen is still there. I can't, however, get the second corrupt frame I suspect is there selected. It is not as bad with `FBox`, though. A double undo fixes the problem without a crash. Multiple undo also gets rid of the corrupt text frame.

Reply:
@jthistle could you please reuse the `isVBoxBase()` from #4705?

Reply:
Will do, thanks.

Reply:
Done @anatoly-os